### PR TITLE
Milestone 4 Desktop Support

### PR DESCRIPTION
**Pull request by eroiyo**
Add Media Queries
  --A lot of CSS rules have been added in order to be readable also on desktop
  --All the sections have been updated

General improvement of the phone version, now readable on any kind of phone

### CODE REVIEW

**Comment by eroiyo** [comment]
The desktop form has more variables than the mobile version, which is normally impossible in a normal situation, so I stick to the mobile version to avoid problems.

File: index.html [hunk 1]
```diff
@@ -48,147 +57,176 @@ <h2>Software Developer</h2>
...
         </div>
       </li>
     </ul>
   </section>
   <section class="contact">
-    <h1>Contact us</h1>
-    <p>If you have an application you are interested in developing, a feature that you need built or a project that
+    <h1 class="contact-title">Contact us</h1>
+    <p class="contact-info">If you have an application you are interested in developing, a feature that you need built
+      or a project that
```
Comment (mricanho) [inline index.html:229]: - [ ] You need to have two inputs for the name. One for the first name and the other for the last name, like the original design
image

File: style.css [hunk 2]
```diff
@@ -218,33 +225,64 @@ li a {
 .work-title {
   display: flex;
   justify-self: flex-start;
-  width: 237px;
+  width: 66.01671309192201vw;
```
Comment (mricanho) [inline style.css:228]: - [ ] All these changes are overdoing it. I understand that you want that the width works with both, the mobile version and the desktop version, but you can go back to the 237px (If that size works with the mobile version) and add a `.work-title { width: //new dimension//` in the media query, that's why we use media queries.

File: style.css [hunk 3]
```diff
@@ -218,33 +225,64 @@ li a {
...
   font-style: normal;
-  font-size: 40px;
-  line-height: 30px;
+  font-size: 11.1vw;
```
Comment (mricanho) [inline style.css:230]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 4]
```diff
@@ -218,33 +225,64 @@ li a {
...
   margin-right: auto;
   color: #9490ff;
-  margin-bottom: 1px;
+  margin-bottom: 1.5vh;
```
Comment (mricanho) [inline style.css:233]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 5]
```diff
@@ -218,33 +225,64 @@ li a {
...
 }
 
 .work-title2 {
   display: flex;
   justify-self: flex-start;
-  width: 237px;
+  width: 66.01671309192201vw;
```
Comment (mricanho) [inline style.css:239]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 6]
```diff
@@ -218,33 +225,64 @@ li a {
...
   font-style: normal;
-  font-size: 40px;
-  line-height: 30px;
+  font-size: 11.1vw;
```
Comment (mricanho) [inline style.css:241]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 7]
```diff
@@ -218,33 +225,64 @@ li a {
...
   margin-right: auto;
   color: #9490ff;
-  margin-top: 10px;
+  margin-top: 1.5vh;
```
Comment (mricanho) [inline style.css:244]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 8]
```diff
@@ -218,33 +225,64 @@ li a {
...
+.down_arrow_a {
+  visibility: hidden;
+  width: 0.0000000000000000000000000001px;
+  margin-right: 10vw;
+}
+
+.down_arrow_b {
+  visibility: hidden;
+  width: 000000000000000000000000000.1px;
 }
```
Comment (mricanho) [inline style.css:268]: - [ ] If the visibility is hidden, you don't need to add anything else

File: style.css [hunk 9]
```diff
@@ -218,33 +225,64 @@ li a {
...
 
 .down_arrow {
-  margin-top: 48px;
+  margin-top: 4.026845637583892vw;
```
Comment (mricanho) [inline style.css:271]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 10]
```diff
@@ -265,21 +315,21 @@ span {
...
   color: black;
-  padding-top: 16px;
-  font-size: 12px;
   transition-duration: 0.4s;
   border: none;
   margin-top: 34px;
   margin-bottom: 40px;
-  width: 30vw;
-  height: 7vh;
+  padding: 5vw;
```
Comment (mricanho) [inline style.css:331]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version. And this is especially problematic if the screen goes up to 1024px, check the google chrome developer tools and experiment with the size.

File: style.css [hunk 11]
```diff
@@ -316,8 +366,8 @@ button:hover {
   top: 129px;
   font-family: "Inter", monospace;
   font-style: normal;
-  font-size: 20px;
-  line-height: 28px;
+  font-size: 4vw;
```
Comment (mricanho) [inline style.css:369]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 12]
```diff
@@ -316,8 +366,8 @@ button:hover {
...
+  line-height: 6vw;
```
Comment (mricanho) [inline style.css:370]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 13]
```diff
@@ -326,17 +376,332 @@ button:hover {
...
+  height: 10vw;
+  border: none;
 }
 
 textarea {
+  resize: none;
   width: 98%;
   margin-top: 4vh;
-  height: 15vh;
+  height: 20vh;
```
Comment (mricanho) [inline style.css:405]: - [ ] Same here, add the new dimension to the media query and let the original dimension for the mobile version

File: style.css [hunk 14]
```diff
@@ -326,17 +376,332 @@ button:hover {
...
+  }
+
+  .work-title,
+  .work-title2 {
+    visibility: hidden;
+    width: 0.000000000000000000000000001px;
+    height: 0.000000000000000000000000001px;
+    margin: 0.000000000000000000000000001px;
+    font-size: 0.00000000000000000000001px;
+  }
```
Comment (mricanho) [inline style.css:520]: - [ ] If the visibility is hidden, you don't need to add anything else

File: style.css [hunk 15]
```diff
@@ -8,7 +8,12 @@ body {
...
-  background-color: #191765;
+  background-color: white;
+}
+
+a {
+  color: white;
+  text-decoration: none;
 }
 
 img {
```
Comment (mricanho) [inline style.css:19]: - [ ] You can add another img in the media query or add a new class in the media query that overwrites this one, right now you have a square and it must be more like a rectangle. Check the grid too, maybe the problem is in there.

Your design:
image

Original design:
image

File: style.css [hunk 16]
```diff
@@ -127,11 +142,13 @@ header {
   width: 100%;
```
Comment (mricanho) [inline style.css:142]: - [ ] The margin-top: 40px is too much for the desktop, you can overwrite it in the media query.

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @eroiyo @Ismailco, 

You did good work following the general requirements. 

**First of all, you need to know that you're going to be exposed to this kind of projects your entire career (if you are going to focus on front-end dev), responsiveness it's super important ([check this article](https://digitalnext.co.uk/dn-hub/blog/importance-of-responsive-web-design/) for more info) and yes, most of the times it's full of pain but it gets better and easier when you practice more. When you say "The desktop form has more variables than the mobile version, which is normally impossible in a normal situation", it's actually not impóssible you just need to use more media queries and the right units to make it through**

**But don't worry, we are here to help**

 ## Required Changes ♻️

- [ ] You have some important issues with all the website, and this is mainly because of the units you are using. The viewport-based units are super good in responsiveness but too much of them can cause more troubles than solutions, the same with percentages. The best way to address the responsiveness is with one or two media queries (in this is specific, one media query) and the use of px and a mix of rem and vh. Please use the google chrome developers tools to check the differences in every size of devices, for example:
Form in Mobile (good)

image

Form in Tablet (Not so good)
image

Form in Desktop 1024px (Not so good)
image

Form in the original design
image

- [ ] You have an overflow in your x-axis, the reason must be a width that it's not working as expected, you need to scan all the widths and check the responsible one.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**It must feel like a lot**
a lot of work

**But if you work hard, you are going to have an excellent and professional portfolio to present to futures companies.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html:226]: Well addressed this!

File: style.css (hunk 2 above)
Comment (DeliceLydia) [reply style.css:234]: I think for now you fixed this but I can suggest that you reduce a font size of `work-title` in the tablet view

File: style.css (hunk 3 above)
Comment (DeliceLydia) [reply style.css:236]: - [ ] Please work on the font size to much each device, you can start with mobile, and as you switch the size of the device check how the `font-size` looks and you can change it from the media queries.

File: style.css (hunk 15 above)
Comment (DeliceLydia) [reply style.css]: Well done this!

File: style.css (hunk 16 above)
Comment (DeliceLydia) [reply style.css:146]: Well addressed this on the desktop!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @eroiyo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, now your desktop view is looking nice,  but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] Check how your mobile view looks like because even if this milestone is for desktop we can't approve it when mobile is not yet ready

pn

please refer to the design and give it some padding or margin to make it look good and to give users the best experience and I suggest that you reduce the font size

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Thanks, I already change the hamburger size, can you check, to see if a nice size?

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved ✔️ 🎉  🎉 
Hi @eroiyo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- [ ] You can add padding or margin between your navbar and the first section on a mobile device and be work on the font size to look good on different device

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**